R/add.R

```diff
@@ -160,26 +221,57 @@ m_add_curve <- function(id, spec = list()) {
 #' @export
 m_add_cylinder <- function(
                            id,
-                           start,
-                           end,
+                           starts,
+                           ends,
```

Comment:
I don't think a change should be required. Just indicate in the documentation that more than one is acceptable is enough.